WordPress/Classes/ViewRelated/Gutenberg/GutenbergMediaPickerHelper.swift

```diff
@@ -2,12 +2,13 @@ import Foundation
 import CoreServices
 import UIKit
 import Photos
+import PhotosUI
 import WordPressShared
 import WPMediaPicker
 import Gutenberg
 import UniformTypeIdentifiers
 
-public typealias GutenbergMediaPickerHelperCallback = ([WPMediaAsset]?) -> Void
+public typealias GutenbergMediaPickerHelperCallback = ([Any]?) -> Void
```

Comment:
I am specifically not using the `WPMediaAsset` protocol because it’s part of the MediaPicker repo and is designed for displaying images in the picker’s collection view cells, but it ended up being used as a marker protocol in multiple places of the app. The goal is to remove the MediaPicker dependency entirely.